roles/register_hypervisor/tasks/main.yml

```diff
@@ -25,6 +25,7 @@
                                       --username='{{ engine_username }}'
                                       --password='{{ admin_password }}'
                                       --host='{{ inventory_hostname }}'
+                                      --host_name='{{ host_name }}'
```

Comment:
Can you make this optional?  Like `{{ "--host_name=" + host_name if host_name is defined else ""}}` or something